indico/modules/events/abstracts/templates/management/abstract.html

```diff
@@ -1,6 +1,144 @@
+                    {% endif %}
+                </div>
+                <div class="abstract-comment-badges">
+                    <span>
+                        {% trans %}Proposes to{% endtrans %}
+                    </span>
+                    {{ _render_review_state(review) }}
+                    {% if review.score %}
+                        {% set score = review.score %}
+                        {# TODO: show max score #}
+                        <span class="i-tag outline highlight" title="{% trans %}Score: {{ score }}/...{% endtrans %}">
+                            {{ score }}
+                        </span>
+                    {% endif %}
+                </div>
+            </div>
+            {% if review.ratings %}
+                {% for rating in review.ratings %}
+                    <div>
+                        {# TODO: UI #}
+                        {{ rating.question.text }}
+                        {{ rating.value }}
+                    </div>
+                {% endfor %}
+                <div class="titled-rule">
+                    {% trans %}Comment{% endtrans %}
+                </div>
+            {% endif %}
+            <div class="i-box-content">
+                {{ review.comment }}
+            </div>
+        </div>
+    </div>
+{% endmacro %}
+
+{% macro _render_decision_box(abstract) %}
+    <div id="abstract-decision-box" class="i-timeline-item">
+        <div class="i-timeline-item-label action icon-hammer"></div>
+        <div class="i-timeline-item-box content-indicator">
+            <div class="i-box-content flex">
+                <div class="stretch">
+                    <strong>
+                        {% trans %}Review summary{% endtrans %}
+                    </strong>
+                </div>
+                <div class="abstract-comment-badges">
+                    {% trans %}Reviews are{% endtrans %}
+                    {{ _render_abstract_reviewing_state(abstract) }}
+                    {% if abstract.score %}
+                        {% set score = abstract.score|round(1) %}
+                        {# TODO: show max score #}
+                        <span class="i-tag highlight" title="{% trans %}Score: {{ score }}/...{% endtrans %}">
+                            {{ score }}
+                        </span>
+                    {% endif %}
+                </div>
+            </div>
+            <div class="i-box-footer">
+                <a href="" class="i-button">
+                    {% trans %}Judge{% endtrans %}
+                </a>
+            </div>
+        </div>
+    </div>
+{% endmacro %}
+
+{% macro _render_review_state(review) %}
+    {% set mapping = {'accept': 'success',
+                      'reject': 'error',
+                      'merge': 'visited',
+                      'mark_as_dupe': 'strong'} %}
+    <span class="i-tag outline {{ mapping[review.proposed_action.name] }}">
+        {{ review.proposed_action.title }}
+        {#- -#}
```

Comment:
But we don't do it explicitly anywhere else except if it's needed because there's no jinja stuff where the `-` can be added